### DISCUSSION TITLE
orderBy possibly broken

### DISCUSSION THREAD

**Discussion by StuartMcD** [solspace/craft-freeform#1275] [Questions & Answers] [answered]
FF 5.1.19.1
CMS 4.9.2

I need to dig deeper but I just had a report of an error after upgrading site to latest versions and did quickly confirm that it's being triggered by the orderBy part of the query, that's all I know this far and have turned off the ordering for now.

I'm unable to get hands on site today so querying whether anyone else can confirm or deny in an effort to possibly eliminate the upgrade.

Everything was working fine prior to upgrade.

**Comment by StuartMcD** [comment]
As a follow up, this is the primary error:

`You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '[[fc19.p_last_name_122]]) subquery
INNER JOIN elements elements ON ... at line 10`

I believe I read somewhere that a DB update or something required when moving to craft V5 - Local setup is using mariadb:10.4 & PHP 8.1

**Reply by StuartMcD** [reply]
Can confirm, a recent update (likely 5.1.19.1) caused orderBy to stop working.

```
{% set playerList = freeform.submissions({
			orderBy: "p_firstName ASC",
			id: ids,
			status: ["pending","offeredByTeam","acceptedByPlayer"],
		}) %}
```

**Comment by StuartMcD** [comment, marked as answer]
Somehow it was working previous  without the` .all()` closing tag D'oH!!

**Reply by kjmartens** [reply]
Thanks for following up. I will reopen and mark this as resolved so others can easily find this solution. 🙂
